### PR TITLE
Doc: format resource document to use the same marks

### PR DESCRIPTION
**Pull request by wuxu92**
i have written a tool to fix the listed issues in the resource document

1. use `*` for the property list mark. (some documents use `-` as a list mark)
2. use a `---` as a separator mark before a block document. (some block has no separator, and some other use `--`)
3. remove redundant `*` before a block document
4. property format should be ``* `name` - (Optional)``, not ``* `name` (Optional) -``
5. A block document must be ``(A|An|The) `xxxx` block supports the following:``
6. must be a `-` after property name, means invalid of ``* `xxx` (Optional)``, must be ``* `xxx` - (Optional)``

The Good:
1. make it easier to analyze our document using program.

### CODE REVIEW

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 1]
```diff
@@ -95,7 +95,7 @@ In addition, one of either `identity` or `service_principal` blocks must be spec
 
 * `auto_scaler_profile` - (Optional) A `auto_scaler_profile` block as defined below.
 
-* `azure_active_directory_role_based_access_control` - (Optional) - A `azure_active_directory_role_based_access_control` block as defined below.
+* `azure_active_directory_role_based_access_control` - - (Optional) A `azure_active_directory_role_based_access_control` block as defined below.
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_cluster.html.markdown:98]: there's a double-dash here?

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 2]
```diff
@@ -127,7 +127,7 @@ In addition, one of either `identity` or `service_principal` blocks must be spec
 
 * `linux_profile` - (Optional) A `linux_profile` block as defined below.
 
-* `local_account_disabled` - (Optional) - If `true` local accounts will be disabled. Defaults to `false`. See [the documentation](https://docs.microsoft.com/azure/aks/managed-aad#disable-local-accounts) for more information.
+* `local_account_disabled` - - (Optional) If `true` local accounts will be disabled. Defaults to `false`. See [the documentation](https://docs.microsoft.com/azure/aks/managed-aad#disable-local-accounts) for more information.
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_cluster.html.markdown:130]: (same here)

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 3]
```diff
@@ -714,6 +714,8 @@ A `web_app_routing` block supports the following:
 
  ---
 
+---
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_cluster.html.markdown:717]: there's now two dividers here?

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hi @wuxu92 

Thanks for this PR. I've taken a look through and left some comments inline, if we can fix those up then we should be able to take another look here and get this merged.

Thanks!

File: website/docs/r/kubernetes_cluster.html.markdown (hunk 3 above)
Comment (wuxu92) [reply website/docs/r/kubernetes_cluster.html.markdown:717]: ha, the already existing divider has a space before it, updated

**Review by katbyte — APPROVED** [review]
LGTM 🔬

**Pull request merged**